cassandra/src/main/scala/akka/stream/alpakka/cassandra/package.scala

```diff
@@ -7,8 +7,8 @@ import com.google.common.util.concurrent.{ FutureCallback, Futures, ListenableFu
 
 import scala.concurrent.{ Future, Promise }
 
-package object cassandra {
-  implicit final class GuavaFutureOpts[A](val guavaFut: ListenableFuture[A]) extends AnyVal {
+package object GuavaFutureOpts {
```

Comment:
Right, thanks. However i still need the `implicit def` the `implicit class` alone doesnt seem to get picked up...